src/GogeToken.sol

```diff
@@ -945,34 +937,37 @@ contract DogeGaySon is ERC20, Ownable {
         uniswapV2Router.swapExactETHForTokensSupportingFeeOnTransferTokens{value: amount}(
             0, // accept any amount of Tokens
             path,
-            deadAddress, // Burn address
+            DEAD_ADDRESS, // Burn address
             block.timestamp.add(300)
         );
 
-        emit SwapBNBForTokens(amount, path);
+        emit BuybackInitiated(amount, path);
```

Comment:
`BuyBackInitiated`?

Reply:
Event emitted when there's a buyback trigged from sells. The buyback mechanism is from the royalties. It uses taxes to buy then burn tokens.